de-lib/src/main/resources/org/iplantc/de/resources/client/messages/IplantErrorStrings.properties

```diff
@@ -49,6 +49,7 @@ folderNotFound = Folder "{0}" not found.
 folderNotReadable = The following folder(s) are not readable:<br />{0}
 folderNotWriteable = The following folder(s) are not writable:<br />{0}
 folderReadable = The following folder(s) are readable:<br />{0}
+folderRefreshFailed = Could not refresh the contents of the folder {0}
```

Comment:
Minor. Possibly change to `.. of the folder: {0}`
or
`Could not refresh the contents of {0}`